### PR TITLE
Update code coverage flow

### PR DESCRIPTION
**Pull request by aangelisc**
Update code coverage workflow to latest version

### CODE REVIEW

File: .github/workflows/code-coverage.yml [hunk 1]
```diff
@@ -4,4 +4,7 @@ on:
     branches-ignore: [dependabot/**]
 jobs:
   workflow-call:
-    uses: grafana/code-coverage/.github/workflows/code-coverage.yml@v0.1.2
+    uses: grafana/code-coverage/.github/workflows/code-coverage.yml@v0.1.6
+    with:
+      frontend-path-regexp: src
+      backend-path-regexp: pkg
```
Comment (andresmgot) [inline .github/workflows/code-coverage.yml:10]: missing a blank line here at the end

**Review by andresmgot — APPROVED** [review]

File: .github/workflows/code-coverage.yml [hunk 2]
```diff
@@ -7,4 +7,5 @@ jobs:
     uses: grafana/code-coverage/.github/workflows/code-coverage.yml@v0.1.6
     with:
       frontend-path-regexp: src
-      backend-path-regexp: pkg
\ No newline at end of file
+      backend-path-regexp: pkg
+
```
Comment (andresmgot) [inline .github/workflows/code-coverage.yml:11]: something is still weird here 😅

Comment (aangelisc) [reply .github/workflows/code-coverage.yml:11]: I'm not sure what's going on 🤔

**Review by andresmgot — APPROVED** [review]
aand it didn't work, but nvm!

**Pull request merged**